web/themes/custom/pineapple/templates/content/node.html.twig

```diff
@@ -86,7 +86,7 @@
   {{ title_prefix }}
   {% if label and not page %}
     {% include "@components/_heading.twig" with {
-      heading_attributes: title_attributes,
+      heading_attributes: title_attributes.addClass('page-title'),
       heading_url: true,
```

Comment:
Heading_url should be a link not Boolean

Reply:
absolutely. Done.